packages/lz4_chans/lz4_chans.3.0.1/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/lz4-chans"
+bug-reports: "https://github.com/UnixJunkie/lz4-chans/issues"
+dev-repo: "git+https://github.com/UnixJunkie/lz4-chans.git"
+license: "BSD-3-Clause"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+# the tests don't pass on OCaml 4.03.0, because of a marshalling behavior
+# that was working before 4.03.0 and put back in place after 4.03.0
+  ["dune" "exec" "-p" name "src/test.exe"] {with-test & ocaml:version != "4.03.0"}
+]
+depends: [
+  "ocaml"
+  "base-unix"
+  "conf-lz4"
+  "dolog" {>= "4.0.0"}
+  "dune"
```

Comment:
```suggestion
  "dune" {>= "1.6"}
```
According to your `dune-project` file, this package requires at least dune 1.6

Reply:
Let's just hope that dune puts the lowest possible version number in there...
Because, the truth is the lowest dune version that can compile this project is OK.
Not just "the dune version that I was using when I generated this file".

Reply:
> Not just "the dune version that I was using when I generated this file".

Dune does that when you just do `dune build` or any dune command that generates it, except for `dune upgrade` which just uses dune 1.0 because dune 1.0 is the closest to jbuilder.